pkg/model/datacatalog/api.go

```diff
@@ -53,3 +53,23 @@ type DeleteAssetResponse struct {
 	// The deletion status
 	Status string `json:"status,omitempty"`
 }
+
+type UpdateAssetRequest struct {
+	// The destination catalog id in which the asset will be updated based on the information provided
```

Comment:
is it catalog ID or asset ID, the comment is not clear

Reply:
changed to assetID. Thanks.